opengrok-web/src/main/webapp/js/utils-0.0.29.js

```diff
@@ -2390,3 +2390,23 @@ function getCookie(cname) {
     }
     return "";
 }
+
+//polyfills for IE10-11
+// References:
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger
+
+
+if (!Number.isInteger)
+{
+    Number.isInteger = function numberIsInteger(value)
+    {
+        return typeof value === 'number' && isFinite(value) && Math.floor(value) === value;
+    };
+}
+
+if (!String.prototype.startsWith) {
+  String.prototype.startsWith = function(searchString, position) {
+    position = position || 0;
+    return this.indexOf(searchString, position) === position;
+  };
```

Comment:
there's only one match, isn't it better to replace it inline?

Reply:
i can change to simple
```
if (event.originalEvent.originalEvent.type.indexOf('key') == 0) {
```